content/docs/user-guide/gems/reference/aws/aws-gamelift/flexmatch/scripting.md

```diff
@@ -0,0 +1,129 @@
+---
+linkTitle: FlexMatch Scripting
+title: FlexMatch Scripting
+description: Learn about Script Canvas nodes for the AWS GameLift Gem in Open 3D Engine (O3DE).
+toc: true
+weight: 400
+---
+
+
+The **AWS GameLift** provides Script Canvas nodes that make requests against Amazon GameLift to start, stop and accept matchmaking requests.
```

Comment:
As a reader, individual screenshots of each node seems only marginally useful to me. Much more valuable, I think, would be a screenshot showing how the nodes might actually be connected, so that the reader can see how the properties are passed in to start matchmaking, for example. Do you think you could come up with a screenshot that demonstrates usage?

Reply:
Same comment for the session management nodes.

Reply:
We do provide some simple examples to use the GameLift gem nodes in practise under the advanced topic section. We don't provide examples for all the nodes since the usage depends on their own game logic and we may not be able to provide Script Canvas graph that makes sense

Reply:
I understand logic will differ from game to game. I'm just thinking about the game designer / script developer who might come to the scripting topic first and see a bunch of screenshots of nodes, but no example how the text description above each one fits with the screenshot. 

For example, the text for StartMatchmaking node reads:
"With this node, you can pass in the properties configuration name, players and ticket id through AWSGameLiftStartMatchmakingRequest."

But the screenshots don't show any of these properties coming into the node (i.e. passed in). And the first 3 screenshots don't even show these properties, and the last 2 screenshots show these properties on the output pins only. So a beginner scripter might be confused how to use these nodes.

Reply:
Sure. Talked with @onecent1101 and one thing we could try is to implement the C++ examples we provided in Script Canvas and that might be helpful.

Reply:
Yes! That'd be perfect. It would be helpful to see the Script Canvas version of the C++ example.